### PR TITLE
open beta announcement blog post

### PR DESCRIPTION
**Pull request by db-vendia**

### CODE REVIEW

File: posts/2021-12-09-announcing-new-features.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+Once you have a [universal application](https://www.vendia.net/product) and a node running in your [Vendia Share account](https://share.vendia.net), you can navigate to the Smart Contracts. When creating a new smart contract, you specify an AWS Lambda Function to execute, the contract's input parameters, which are specified as GraphQL queries, and the output parameters, which are specified as GraphQL mutations. When a Smart Contract is triggered, the input query is executed and the returned value forms the arguments passed to the Lambda Function. The result of the Lambda Function is captured in the output GraphQL mutation, which is used to update the data based on the function's output.
+
+![](https://user-images.githubusercontent.com/92179243/145060082-56557805-7708-4a26-8818-8dcfb95ceb78.png "smart contracts")
```
Comment (brian-vendia) [inline posts/2021-12-09-announcing-new-features.md:33]:
```suggestion
![smart contracts](https://user-images.githubusercontent.com/92179243/145060082-56557805-7708-4a26-8818-8dcfb95ceb78.png)
```

File: posts/2021-12-09-announcing-new-features.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ 
+**Enable More Users with No Code, Entity Explorer View**
+
+We also updated our User Interface (UI) to enable non-developers to create and manage decentralized data applications (called Universal Applications). A new simple workflow guides users through the steps to quickly create a new Universal Application. The new UI also includes an Entity Explorer view allowing non-developers to read from and write to a Universal Applications without crafting GraphQL queries or mutations. 
+
+Similar to Smart Contracts, you can navigate to the Entity Explorer from within a node. Within the Entity Explore you can see all the tables from your data model in the left column with data in the selected table shown. 
+
+![](https://user-images.githubusercontent.com/92179243/145060323-52c8b712-acc6-4fea-8a03-eff794849c63.png "Entity Explorer")
```
Comment (brian-vendia) [inline posts/2021-12-09-announcing-new-features.md:41]:
```suggestion
![Entity Explorer](https://user-images.githubusercontent.com/92179243/145060323-52c8b712-acc6-4fea-8a03-eff794849c63.png)
```

File: posts/2021-12-09-announcing-new-features.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+Clicking into the ID for any of the rows, will open the ability to edit, delete or view the full history of that entry. You have full control over which users can view, edit or delete data through the Entity Explorer. 
+
+![](https://user-images.githubusercontent.com/92179243/145060474-7b676775-746e-4493-95be-adb37ce1f059.png "Edit a entity explorer")
```
Comment (brian-vendia) [inline posts/2021-12-09-announcing-new-features.md:45]:
```suggestion
![Edit an entity](https://user-images.githubusercontent.com/92179243/145060474-7b676775-746e-4493-95be-adb37ce1f059.png)
```

File: posts/2021-12-09-announcing-new-features.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+**Connect more partners with Azure support**
+
+In addition to the existing Amazon Web Services support, Vendia Share now includes beta support for Microsoft Azure, making it easy for partners to connect and develop from their Azure accounts. Customers select the appropriate Azure region to ensure low latency data transfer and Vendia Share handles provisioning and managing the resources automatically. This new functionality allows users to create integrations to Azure Queues, Azure Functions, and other Azure-specific services through the use of Azure Event Grid. For example, a user may want to execute a specific Azure Function whenever another participant with access to an AWS node creates a new record. Azure support is currently in beta and available for all customers. 
+
+![](https://user-images.githubusercontent.com/92179243/145244534-292321e8-19a4-4e04-90cf-d7c9ef8143e4.png)
```
Comment (brian-vendia) [inline posts/2021-12-09-announcing-new-features.md:23]:
```suggestion
![Azure node](https://user-images.githubusercontent.com/92179243/145244534-292321e8-19a4-4e04-90cf-d7c9ef8143e4.png)
```

File: posts/2021-12-09-announcing-new-features.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+ 
+**Enable More Users with No Code, Entity Explorer View**
+
+We also updated our User Interface (UI) to enable non-developers to create and manage decentralized data applications (called Universal Applications). A new simple workflow guides users through the steps to quickly create a new Universal Application. The new UI also includes an Entity Explorer view allowing non-developers to read from and write to a Universal Applications without crafting GraphQL queries or mutations. 
+
+Similar to Smart Contracts, you can navigate to the Entity Explorer from within a node. Within the Entity Explore you can see all the tables from your data model in the left column with data in the selected table shown. 
+
+![](https://user-images.githubusercontent.com/92179243/145060323-52c8b712-acc6-4fea-8a03-eff794849c63.png "Entity Explorer")
+
+Clicking into the ID for any of the rows, will open the ability to edit, delete or view the full history of that entry. You have full control over which users can view, edit or delete data through the Entity Explorer.
```
Comment (brian-vendia) [inline posts/2021-12-09-announcing-new-features.md:43]:
```suggestion
Clicking into the `_id` for any of the rows, will open the ability to edit, delete or view the full history of that entry. You have full control over which users can view, edit or delete data through the Entity Explorer. 
```

File: posts/2021-12-09-announcing-new-features.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+Clicking into the ID for any of the rows, will open the ability to edit, delete or view the full history of that entry. You have full control over which users can view, edit or delete data through the Entity Explorer. 
+
+![](https://user-images.githubusercontent.com/92179243/145060474-7b676775-746e-4493-95be-adb37ce1f059.png "Edit a entity explorer")
+
+**Grow with New Pricing Plans**
+
+We are also announcing that Vendia Share is now available as a SaaS offering, with pricing that scales from single users through large enterprises. All pricing plans include the new features, along with the established capabilities for building and running highly scalable, fault-tolerant, cross-cloud, multi-account, multi-region, multi-party applications. Share is available in three plans: Starter, Individual, and Enterprise. Starter is our free plan that offers developers generous limits to test Vendia at no cost. The Individual plan removes most of the limits of the Starter plan and starts at $99 per month. The Starter and Individual plans are currently in beta and available for all customers. For more information, see the [pricing page](https://www.vendia.net/pricing).
```
Comment (brian-vendia) [inline posts/2021-12-09-announcing-new-features.md:49]:
```suggestion
We are also announcing pricing that meets customers' demand and allows them to scale their usage from single users through large enterprises. All pricing plans include the new features, along with the established capabilities for building and running highly scalable, fault-tolerant, cross-cloud, multi-account, multi-region, multi-party applications. Share is available in three plans: Starter, Individual, and Enterprise. Starter is our free plan that offers developers generous limits to test Vendia at no cost. The Individual plan removes most of the limits of the Starter plan and starts at $99 per month. The Starter and Individual plans are currently in beta and available for all customers. For more information, see the [pricing page](https://www.vendia.net/pricing). 
```

**Review by brian-vendia — CHANGES_REQUESTED** [review]
I really like where this is headed!

**Comment by brian-vendia** [comment]
@db-vendia please talk with Kevin regarding the release date. I'm not sure when this post should go out so you should adjust the publish date accordingly.

**Review by james-vendia — APPROVED** [review]
Looks good!

**Pull request merged**